operator/validator/controller.go

```diff
@@ -1358,8 +1358,10 @@ func SetupCommitteeRunners(
 		}
 		config.ValueCheckF = valueCheckF
 
-		identifier := spectypes.NewMsgID(options.NetworkConfig.DomainType(), options.Operator.CommitteeID[:], role)
-		qbftCtrl := qbftcontroller.NewController(identifier[:], options.Operator, config, options.OperatorSigner, options.FullNode)
+		qbftCtrl := qbftcontroller.NewController(func() []byte {
+			identifier := spectypes.NewMsgID(options.NetworkConfig.AlanDomainType, options.Operator.CommitteeID[:], role)
+			return identifier[:]
+		}, options.Operator, config, options.OperatorSigner, options.FullNode)
```

Comment:
this would be clearer to read:
```suggestion
		identifier := func() []byte {
			identifier := spectypes.NewMsgID(options.NetworkConfig.AlanDomainType, options.Operator.CommitteeID[:], role)
			return identifier[:]
		}
		qbftCtrl := qbftcontroller.NewController(identifier, options.Operator, config, options.OperatorSigner, options.FullNode)
```